### ISSUE TITLE
Log files not opening from SCP.

### ISSUE THREAD

**Issue by klonos** [WPN-XM/WPN-XM#356]
None of the logs open. There's only a "Warning, log file not found" for php_error.log message.

**Comment by klonos** [comment]
...my logs folder contains the following log files (plus the setup log file):

```
access.log
error.log
mariadb_error.log
mongodb.log
nginx.pid
postgresql.log
```

**Comment by jakoch** [comment]
Strange. All log files are there.
The log buttons are forwarding to the configured editor.
By default it's notepad.exe.
```
 settings->set("global/editor",                   "notepad.exe");
```

**Comment by klonos** [comment]
The default was set to just ```notepad.exe```. I've changed this to ```C:\Windows\notepad.exe``` (using the SCP -> Configuration -> Select Editor option) and it now works.

**Comment by jakoch** [comment]
notepad.exe should work, too - because the Windows folder is always on the environment PATH.

**Comment by klonos** [comment]
Confirming fixed in 0.8.2. Thanx!
